lint.go

```diff
@@ -1038,22 +1038,38 @@ func (f *file) lintRanges() {
 	})
 }
 
-// lintErrorf examines errors.New calls. It complains if its only argument is an fmt.Sprintf invocation.
+// lintErrorf examines errors.New and testing.Error calls. It complains if their only arguments are fmt.Sprintf invocations.
 func (f *file) lintErrorf() {
+	testingTName := ""
 	f.walk(func(node ast.Node) bool {
-		ce, ok := node.(*ast.CallExpr)
-		if !ok {
-			return true
-		}
-		if !isPkgDot(ce.Fun, "errors", "New") || len(ce.Args) != 1 {
-			return true
-		}
-		arg := ce.Args[0]
-		ce, ok = arg.(*ast.CallExpr)
-		if !ok || !isPkgDot(ce.Fun, "fmt", "Sprintf") {
-			return true
+		switch ce := node.(type) {
```

Comment:
`ce` was an abbreviation of `CallExpr`, so it doesn't make sense to retain that name now. Use `v`.